### PR TITLE
Add synthetic KLV test videos

### PR DESCRIPTION
**Pull request by daniel-riehm**
This PR adds five 30-frame test videos with comprehensive ST0601 test data, and FFmpeg input tests to ensure there are no regressions as KLV development continues. Also included is a tweak to the JSON reader to make it report the stream index of the packet when reading, which is necessary for one of the tests.

The data file sizes are interesting here: each video is below 200KB, but the two JSON reference files are 2MB each. Reviewers may opine whether these sizes are a problem; the JSON files could be reduced significantly by removing whitespace and unnecessary field labels, albeit at the cost of readability.

@hdefazio

### CODE REVIEW

**Comment by hdefazio** [comment]
I think during the weekly meeting we decided to zip the large json files?

**Comment by daniel-riehm** [comment]
I've pushed up changes which store the ground-truth JSON files as compressed versions to bring them under the 1MB file size limit, so the tests will now decompress those when reading them in.

**Comment by daniel-riehm** [comment]
Windows failure looks unrelate - something in MVG.

**Comment by daniel-riehm** [comment]
jenkins test this please

**Comment by daniel-riehm** [comment]
jenkins test this please

File: arrows/ffmpeg/tests/test_video_input_ffmpeg_klv.cxx [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+  verify_standard_video();
+}
+
+// ----------------------------------------------------------------------------
+TEST_F ( ffmpeg_video_input_klv, h264_no_klv_verify )
+{
+  input.open( no_streams_path );
+
+  kv::timestamp ts;
+  for( size_t i = 0; i < 30; ++i )
```
Comment (chetnieter) [inline arrows/ffmpeg/tests/test_video_input_ffmpeg_klv.cxx:273]: Number of frames common to two tests. Define once?

**Review by chetnieter — CHANGES_REQUESTED** [review]
Looks good - just some minor nit-picks. Given the the json files are compressed I don't think their size is a problem.

**Comment by daniel-riehm** [comment]
Now that the MVG issue has been resolved, looks like there is an actual test failure on Windows related to this PR...

**Comment by daniel-riehm** [comment]
@chetnieter Are there any further issues?

**Review by chetnieter — APPROVED** [review]

**Pull request merged**